woocommerce/woocommerce-gateway-stripe
# fix: UPE note when account disconnected

# Changes proposed in this Pull Request:
Let's not display the UPE inbox note if the Stripe account is not connected.

# Testing instructions
- Ensure your account is connected (keys for should be entered in order to be in that state)
- Ensure the Stripe gateway is enabled
- Ensure you have the `_wcstripe_feature_upe_settings` flag enabled
- The note should be displayed
